Core/src/org/sleuthkit/autopsy/contentviewers/SQLiteViewer.java

```diff
@@ -414,28 +435,28 @@ private boolean findAndCopySQLiteMetaFile(AbstractFile sqliteFile, String metaFi
      *
      * @return true if success, false otherwise
      */
-    private boolean getTables() {
-
-        try {
-            Statement statement = connection.createStatement();
+    private void getTables() throws SQLException {
 
-            ResultSet resultSet = statement.executeQuery(
-                    "SELECT name, sql FROM sqlite_master "
-                    + " WHERE type= 'table' "
-                    + " ORDER BY name;"); //NON-NLS
+        
+        Statement statement = connection.createStatement();
+        ResultSet resultSet = statement.executeQuery(
+                "SELECT name, sql FROM sqlite_master "
+                + " WHERE type= 'table' "
+                + " ORDER BY name;"); //NON-NLS
 
-            while (resultSet.next()) {
-                String tableName = resultSet.getString("name"); //NON-NLS
-                String tableSQL = resultSet.getString("sql"); //NON-NLS
+        while (resultSet.next()) {
+            String tableName = resultSet.getString("name"); //NON-NLS
+            String tableSQL = resultSet.getString("sql"); //NON-NLS
 
-                dbTablesMap.put(tableName, tableSQL);
-            }
-        } catch (SQLException e) {
-            LOGGER.log(Level.SEVERE, "Error getting table names from the DB", e); //NON-NLS
+            dbTablesMap.put(tableName, tableSQL);
         }
-        return true;
+        resultSet.close();
+	statement.close();
```

Comment:
I think a finally block is needed here to ensure that the record set and the statement are always closed.

Reply:
Addressed.